test/bitmap-header.test.js

```diff
@@ -1,30 +1,36 @@
+// eslint-disable-next-line no-unused-vars
 const assert = require('assert');
 const constants = require('../lib/bitmap-constants');
 const BitmapHeader = require('../lib/bitmap-header');
+const fs = require('fs');
 
 describe('bitmap header', () => {
 
-    let buffer = null;
-    beforeEach(() => {
-        // TODO: file read './test/test-bitmap.bmp' and put the promise return into buffer variable
+  let buffer = null;
+  beforeEach(done => {
+    // TODO: file read './test/test-bitmap.bmp' and put the promise return into buffer variable
+    fs.readFile('./test/test-bitmap.bmp', (err, data) => {
+      buffer = data;
+      done();
     });
+  });
 
-    it('has correct specs', () => {
-        // TODO: read the wiki spec docs to figure out what these values should be.
-        // You don't need to change this test, you need to put the correct values into
-        // '../lib/bitmap-constants'
-        assert.ok(constants.PIXEL_OFFSET);
-        assert.ok(constants.BITS_PER_PIXEL_OFFSET);
-        assert.ok(constants.FILE_SIZE_OFFSET);
-    });
+  it('has correct specs', () => {
+    // TODO: read the wiki spec docs to figure out what these values should be.
+    // You don't need to change this test, you need to put the correct values into
+    // '../lib/bitmap-constants'
+    expect(constants.PIXEL_OFFSET);
```

Comment:
Per the Jest docs: "You will rarely call expect by itself. Instead, you will use expect along with a "matcher" function to assert something about a value." In this case, you could use a matcher like so:
```suggestion
    expect(constants.PIXEL_OFFSET).toBeDefined();
```